### DISCUSSION TITLE
Feature Request: 'sales-pop' multiple new features idea

### DISCUSSION THREAD

**Discussion by hz-tyfoon** [getdokan/storegrowth-sales-booster#425] [Feature Request]
### What:
We can add multiple features in our 'sales-pop' module. here's some screenshots:

### Screenshots:
<img width="1488" alt="Screenshot at Jul 16 18-11-50" src="https://github.com/Invizo/storegrowth-sales-booster/assets/59374562/f6ea6ae3-56df-4d63-8735-ea5fa5a81c53">
-----
<img width="1651" alt="Screenshot at Jul 16 18-12-42" src="https://github.com/Invizo/storegrowth-sales-booster/assets/59374562/6ac19475-7e8c-4dd2-96e8-77b0e7ad00a9">
-----
<img width="1469" alt="Screenshot at Jul 16 18-13-06" src="https://github.com/Invizo/storegrowth-sales-booster/assets/59374562/5c0a9351-cb9a-4bd3-8688-e7c75c4725cf">
-----
<img width="1636" alt="Screenshot at Jul 16 18-13-55" src="https://github.com/Invizo/storegrowth-sales-booster/assets/59374562/e9807c2c-da49-494c-a793-5b00d3315132">
-----
<img width="1466" alt="Screenshot at Jul 16 18-14-20" src="https://github.com/Invizo/storegrowth-sales-booster/assets/59374562/b0d6bcce-aba3-46c8-8b28-b914913b5b93">
-----
<img width="1479" alt="Screenshot at Jul 16 18-14-35" src="https://github.com/Invizo/storegrowth-sales-booster/assets/59374562/2d8b7c03-64ff-4b51-9451-8603066620fd">
-----
<img width="1623" alt="Screenshot at Jul 16 18-15-03" src="https://github.com/Invizo/storegrowth-sales-booster/assets/59374562/1ed5e44a-99b7-4020-828b-6e5f083fdcf3">
-----
<img width="1495" alt="Screenshot at Jul 16 18-15-26" src="https://github.com/Invizo/storegrowth-sales-booster/assets/59374562/5e11174a-9b39-44cf-a37f-d8d04ac2fbe5">
-----
<img width="1496" alt="Screenshot at Jul 16 18-15-41" src="https://github.com/Invizo/storegrowth-sales-booster/assets/59374562/998faff1-8d1e-421c-96d1-cd2c1e951bf2">
-----
<img width="1348" alt="Screenshot at Jul 16 18-15-55" src="https://github.com/Invizo/storegrowth-sales-booster/assets/59374562/fd2ea85d-a551-4135-9a83-8794fa5ac2ad">
-----
<img width="1089" alt="Screenshot at Jul 16 18-16-07" src="https://github.com/Invizo/storegrowth-sales-booster/assets/59374562/d6a3a1bf-eb58-4fd9-884c-9d592aa3d0af">
-----
<img width="1058" alt="Screenshot at Jul 16 18-16-20" src="https://github.com/Invizo/storegrowth-sales-booster/assets/59374562/caa4f630-9b5e-4621-8e75-d3b249a38462">
-----
<img width="1044" alt="Screenshot at Jul 16 18-16-31" src="https://github.com/Invizo/storegrowth-sales-booster/assets/59374562/4b3ba78c-9930-4286-b58b-46432dfa4313">

**Comment by imtiaz-pranto** [comment]
Converting it into discussion for further processing
